### PR TITLE
support log roller by configuration

### PR DESCRIPTION
**Pull request by taoyuanyuan**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
1. support log roller by configuration
2. change logtime from s to ms

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/log/errorlog.go [hunk 1]
```diff
@@ -37,7 +38,7 @@ type errorLogger struct {
 }
 
 func CreateDefaultErrorLogger(output string, level Level) (ErrorLogger, error) {
-	lg, err := GetOrCreateLogger(output)
+	lg, err := GetOrCreateLogger(output, nil)
```
Comment (nejisama) [inline pkg/log/errorlog.go]: DefaultLogger 不支持配置么？现在好像就一个DefaultLogger

File: pkg/log/logger_test.go [hunk 2]
```diff
@@ -29,7 +27,7 @@ import (
 )
```
Comment (nejisama) [inline pkg/log/logger_test.go]: 感觉可以补一个配置了Roller的UT？

File: pkg/log/logger.go [hunk 3]
```diff
@@ -80,13 +80,18 @@ type Logger struct {
 // key is output, same output reference the same Logger
 var loggers sync.Map // map[string]*Logger
 
-func GetOrCreateLogger(output string) (*Logger, error) {
+func GetOrCreateLogger(output string, roller *Roller) (*Logger, error) {
```
Comment (nejisama) [inline pkg/log/logger.go]: 没有看到对应的配置改动？只能是代码里改？
